apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/FragmentsResponseMapperBuilder.kt

```diff
@@ -8,7 +8,7 @@ import com.squareup.javapoet.*
 import javax.lang.model.element.Modifier
 
 /**
- * Responsible for [Fragments.Mapper] class generation
+ * Responsible for Fragments.Mapper class generation
```

Comment:
Didn't really understand what is `Fragments.Mapper` here. Since there is not such class, Kotlin does not like it.